### DISCUSSION TITLE
timeouts plugin for old versions

### DISCUSSION THREAD

**Discussion by mo-saeed** [coredns/coredns#5913] [General]
Hi,

would it be possible to add the timeout plugin to version 1.9.3 ?

**Comment by chrisohaver** [comment]
It should be possible for you to back port it and build a custom image.  It would probably be easier to just use the latest version though.  IIRC, I don't think there were any deprecations between 1.9 and 1.10.

If you are asking if the CoreDNS project will retroactively add it to 1.9.3: No, this will not happen for a version that has already been released.
